### ISSUE TITLE
Tutorial / Adding routes

### ISSUE THREAD

**Issue by blondie63** [hapijs/hapijs.com#535]
On page https://hapijs.com/tutorials the sample code under "Adding routes" does'nt work with Hapi V17 !
err: Server running at: http://localhost:3000
Debug: internal, implementation, error
    TypeError: reply is not a function

What's wrong ?

**Comment by wswoodruff** [comment]
@blondie63 Yo! Hapi's tutorials page right now is only compatible with `hapi v16.x.x` (says _This tutorial is compatible with hapi v16_  at the top of the page) -- while this doesn't necessarily raise flags to a newcomer, the latest hapi release, `v17` has a lot of breaking changes.

Please bear with us as the hapi community is working on updating the tutorials page.

For your particular case, checkout the example here in the v17 docs: https://github.com/hapijs/hapi/blob/master/API.md#server.route()

**Comment by wswoodruff** [comment]
Specifically, the `server.connection` function is gone, just declare anything that would've gone in there to the main hapi server.

Also, we don't return with `reply` in v17. The reply interface has been replaced with a [response toolkit](https://github.com/hapijs/hapi/blob/master/API.md#response-toolkit).

Lastly, to reply with values in your routes, just return them as in the example for `server.route()` I linked above.
